spec/models/reaction_destroy_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe Reaction, type: :model do
+  let(:article) { create(:article, featured: true) }
+  let!(:reaction) { build(:reaction, reactable: article) }
+
+  before do
+    Delayed::Worker.delay_jobs = true
+  end
+
+  after do
+    Delayed::Worker.delay_jobs = false
+  end
+
+  it "creates a ScoreCalcJob on article reaction destroy" do
+    expect do
+      reaction.destroy
+    end.to change(Delayed::Job.where(queue: "articles_score_calc"), :count).by(1)
```

Comment:
Because `Articles::ScoreCalcJob` is an active job, it makes more sense for rspec to interface with ActiveJob instead of Delayed job.

```suggestion
    ActiveJob::Base.queue_adapter = :test
    expect { reaction.destroy }.to have_enqueued_job(Articles::ScoreCalcJob).exactly(:once)
```
you'll want to remove the before/after hook and might be worth it to move this spec in to `reaction_spec`. This can also be applied to `spec/models/article_destroy_spec.rb`

It would also be worthwhile to generate [job specs](https://relishapp.com/rspec/rspec-rails/v/3-8/docs/job-specs/job-spec) for these two new job classes.